src/web/rustdoc.rs

```diff
@@ -367,6 +368,11 @@ pub fn rustdoc_html_server_handler(req: &mut Request) -> IronResult<Response> {
     // Get the latest version of the crate
     let latest_version = latest_release.version.to_string();
     let is_latest_version = latest_version == version;
+    let is_prerelease = semver::Version::parse(&version)
+        // should be impossible unless there is a semver incompatible version in the db
+        // Note that there is a redirect earlier for semver matches to the exact version
+        .map_err(|_| IronError::new(Nope::InternalServerError, status::InternalServerError))?
```

Comment:
Can you add a log message here that says the error was invalid semver? That would help us notice these bugs.